spec/features/supplier_login_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+feature "Supplier Login" do
+  let(:default_supplier_attributes) do
+    { name: "Supplier X",
+      slug: "supplier-x",
+      description: "A store for all of you disaster relief needs",
+      retired: false,
+      address: "101 William White Blvd, Pueblo, Co 80111" }
+    end
```

Comment:
`end` at 10, 4 is not aligned with `let(:default_supplier_attributes) do` at 4, 2